fvtest/porttest/si.cpp

```diff
@@ -2036,3 +2036,29 @@ TEST(PortSysinfoTest, sysinfo_test_get_open_file_count)
 }
 #endif /* defined(LINUX) || defined(AIXPPC) */
 #endif /* !(defined(WIN32) || defined(WIN64)) */
+
+/**
+ * Test omrsysinfo_test_get_os_description.
+ */
+TEST(PortSysinfoTest, sysinfo_test_get_os_description)
+{
+	OMRPORT_ACCESS_FROM_OMRPORT(portTestEnv->getPortLibrary());
+	const char *testName = "omrsysinfo_test_get_os_description";
+
+	int32_t rc = 0;
+	reportTestEntry(OMRPORTLIB, testName);
+
+	OMROSDesc desc = {0};
+	rc =  omrsysinfo_get_os_description(&desc);
+
+	for (int i = 0; i < OMRPORT_SYSINFO_OS_FEATURES_SIZE * 32; i++) {
+		bool feature = omrsysinfo_os_has_feature(&desc, i);
+		if ((true != feature) && (false != feature)) {
```

Comment:
yes ...
Changed bool -> BOOLEAN, true -> TRUE and false -> FALSE.
Verifying if it is a valid BOOLEAN.